tests/system-kmod-macros.at

```diff
@@ -185,3 +185,8 @@ m4_define([OVS_CHECK_KERNEL_EXCL],
     sublevel=$(uname -r | sed -e 's/\./ /g' | awk '{print $ 2}')
     AT_SKIP_IF([ ! ( test $version -lt $1 || ( test $version -eq $1 && test $sublevel -lt $2 ) || test $version -gt $3 || ( test $version -eq $3 && test $sublevel -gt $4 ) ) ])
 ])
+
+m4_define([CHECK_SMALL_V6_FRAG],
+[
+    OVS_CHECK_KERNEL_EXCL(4, 19, 4, 20)
```

Comment:
Should we not exclude all versions >= 4.19 ?

This could be done using some large value for max version such as 1000, 0 to approximate 'unlimited'
or
using a special 'unlimited' indicator such as 0, 0 and then adding a check for that value in OVS_CHECK_KERNEL_EXCL

I think we should add a comment to CHECK_SMALL_V6_FRAG stating the change in kernel behavior to have
a hardcoded lower limit enforced.